modules/openapi-generator-maven-plugin/README.md

```diff
@@ -101,7 +106,7 @@ Specifying a custom generator is a bit different. It doesn't support the classpa
             </goals>
             <configuration>
                 <inputSpec>${project.basedir}/src/main/resources/yaml/yamlfilename.yaml</inputSpec>
-                <!-- language file, like e.g. JavaJaxRSCodegen -->
+                <!- language file, like e.g. JavaJaxRSCodegen -->
```

Comment:
@InfoSec812 the dash seems to be accidentally removed?

Reply:
Whoops! Good catch. Must have accidentally done that while doing a find/replace to update the docs. Ill fix it now.